src/main/java/net/sf/jabref/logic/util/strings/StringUtil.java

```diff
@@ -669,4 +669,32 @@ public static String expandAuthorInitials(String name) {
         return sb.toString().trim();
     }
 
+    /**
+     * Return a String with n spaces
+     *
+     * @param n Number of spaces
+     * @return String with n spaces
+     */
+    public static String nSpaces(int n) {
+        return nCharacters(n, ' ');
+    }
+
+    /**
+     * Return a String with n copies of the char c
+     *
+     * @param n Number of copies
+     * @param c char to copy
+     * @return String with n copies of c
+     */
+    public static String nCharacters(int n, char c) {
```

Comment:
I've seen references to it when googling, but wasn't sure which libraries we already use (there's also one in Apache StringUtils).